modules/custom/az_core/az_core.info.yml

```diff
@@ -10,4 +10,5 @@ dependencies:
   - config_sync:config_sync
   - config_update:config_update
   - drupal:telephone
+  - linkit:linkit
```

Comment:
Do we want this here or in the az_media.info.yml, My preference is az_media, since that is where the az_standard formatter lives.

Reply:
The text editor that uses linkit lives in az_paragraphs.

Reply:
I thought this was the base format https://github.com/az-digital/az_quickstart/pull/1836/files#diff-8fd46582dff144201c01683808b026028abb0b645432085f71d49529323cf213R90-R96